lib/VoxeetMedia.ts

```diff
@@ -1,27 +1,4 @@
-/*
```

Comment:
Is it expected to remove the header?

Reply:
The license in the header is not the same as the license of the repo. Plus, it is not in the react native library.